src/Symfony/Component/Workflow/WorkflowInterface.php

```diff
@@ -37,11 +37,17 @@ public function getMarking($subject);
      * @param string $transitionName A transition
      *
      * @return bool true if the transition is enabled
-     *
-     * @throws LogicException
      */
     public function can($subject, $transitionName);
 
+    /**
+     * Build a TransitionBlockerList to know why a transition is blocked.
+     *
+     * @param object $subject        A subject
+     * @param string $transitionName A transition
+     */
+    public function buildTransitionBlockerList($subject, $transitionName): TransitionBlockerList;
```

Comment:
As this is a new method, you can use type hints here at least for `$transitionName`